.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install kubectl
+      run: |
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
+        sudo chmod +x /usr/bin/kubectl
+
+    - name: Install krew
+      shell: bash
+      run: |
+        cd "$(mktemp -d)" &&
+        curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/${KREW_VERSION}/krew.{tar.gz,yaml}" &&
+        tar zxvf krew.tar.gz &&
+        ./krew-linux_amd64 install --manifest=krew.yaml --archive=krew.tar.gz
+        
+    - name: Validate updated plugin manifests
+      run: |
+        set -u
```

Comment:
I honestly don't know why don't you just add `set -euo pipefail` which we discussed before.

Imagine `git diff` failed. This `run` shell will NOT fail, because `git` is not the last command in the pipe. No data will flow to `xargs`, so it will succeed, which will make us think everything worked.

Reply:
The `shell: bash` setting enables `set -eo pipefail`. However since it doesn't enable `-u` which we also want, I'd recommend to not use `shell: bash` and instead be very explicit about `set -euo pipefail` in all scripts.

Reply:
I think we can keep `shell: bash` (I assume that is the default), and still `set -euo pipefail` instead of relying on Github to pull the rug under us in the future.

Reply:
I fully agree. And you're right, `bash` is the default shell.

Reply:
it's done you can check it.